src/openforms/js/components/admin/form_design/registrations/objectsapi/LegacyConfigFields.js

```diff
@@ -132,7 +132,7 @@ JSONTemplateField.propTypes = {
   helpText: PropTypes.node,
 };
 
-const ContentJSON = () => (
+export const ContentJSON = () => (
```

Comment:
re-using this here as-is because it's legacy - I'm hoping we can remove all of this in 3.0 :)

Reply:
I think for 3.0 we need to either remove the objects API integration in ZGW APIs (this was a temporary solution for Utrecht if I remember correctly), or convert it to the Objects API variable mapping approach. Either way, the `contentJson` template is gone in 3.0. That's why I didn't find it worth it to move this to a shared module instead, considering it should only exist here for another 3 months and then it's gone.